src/main/scala/org/phenoscape/kb/queries/DirectPhenotypesForTaxon.scala

```diff
@@ -58,11 +59,13 @@ object DirectPhenotypesForTaxon {
       SELECT DISTINCT ?state ?description ?matrix ?matrix_label ?phenotype ?character ?character_label
       FROM $KBMainGraph
       FROM $KBClosureGraph
+      FROM $KBRedundantRelationGraph
       $namedQueriesBlock
       $whereClause
       ORDER BY LCASE(?description) ?phenotype
       $paging
       """
+      print("chutki \n" + query.text)
```

Comment:
Don't forget to remove.

Reply:
Yeah, I will remove it once this draft PR gets finalized.